tasks/averages/simple-moving-average/src/main.rs

```diff
@@ -10,7 +10,7 @@ struct MovingAverageError {
 
 impl MovingAverage {
     fn new(period: u32) -> MovingAverage {
-        if period == 0 {
```

Comment:
We want the constructor to panic if the period is equal to 0. In the old code, it can only be constructed if the period is equal to 0.